### PR TITLE
Added LPM and reset support to sfputil for MLNX msn2700 platform [NOT FOR master]

### PR DESCRIPTION
**Pull request by andrii-savka**
**For review ONLY**
**PR can be merged after integrating MLNX SDK version 4.2.7000X042**
**Also the support for other MLNX platform will be added very soon**
**Currently support is available only for SN2700** 

**- What I did**
Added LPM and reset support to sfputil for MLNX platforms 

**-How I did it**
Implemented additional scripts which uses a MLNX SDK

**- How to verify it**
1. For reset - check the admin status during restart
2. For LPM:
2.1 Get LPM status
2.2 Set LPM to OFF
2.3 Get LPM status and check if it's OFF
2.4 Set LPM to ON
2.5 Get LPM status and check if it's ON

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmget.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmget.py]: `LMP ON` appears to be a typo. Should be `LPM ON`.

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmset.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmset.py]: `LMP` should be `LPM`

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfpreset.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfpreset.py]: `LMP` should be `LPM`

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: `LMP ON` should be `LPM ON` (see my review comment on sfplpmget.py above).

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: `LMP` should be `LPM`

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: `LMP` should be `LPM`

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (jleveque) [inline device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: `LMP` should be `LPM`

**Review by jleveque — CHANGES_REQUESTED** [review]

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmget.py
Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmget.py]: The typo is fixed.

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmset.py
Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfplpmset.py]: The typo is fixed. The typo is fixed.

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfpreset.py
Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfpreset.py]: The typo is fixed.

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: The typo is fixed.

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: The typo is fixed.

File: device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py
Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: The typo is fixed.

Comment (andrii-savka) [reply device/mellanox/x86_64-mlnx_msn2700-r0/plugins/sfputil.py]: The typo is fixed.

**Review by jleveque — APPROVED** [review]

**Comment by andrii-savka** [comment]
It can be merged after #13

**Comment by andrii-savka** [comment]
New issue with LPM setting is found. Don't merge this PR without trigger.

**Comment by andrii-savka** [comment]
PR #10 can be merged after #13

**Review by jleveque — APPROVED** [review]

**Comment by andrii-savka** [comment]
@lguohan @jleveque 
Rebased to the latest v1.2

**Pull request merged**
